hub/powertoys/run.md

```diff
@@ -105,46 +136,119 @@ If the program plugin's option "Include in global result" is not selected, be su
 
 `.Visual Studio Code -- C:\myFolder`
 
-## Calculator Plugin
+### Calculator Plugin
+
+> [!NOTE]
+> The calculator plugin respects your computers number format setting. Please be aware of the different decimal delimiters in different languages when searching.
+
+> [!WARNING]
+> There is a known issue that the comma sign used in some operations as delimiter between numbers gets interpreted as decimal delimiter. This happens if your number format setting in Windows is configured to use the comma sign as decimal separator.
 
 The PowerToys Run calculator plugin supports the following operations:
 
-| Operation |  Operator Syntax |
-| - | - |
-| Addition |  a + b |
-| Subtraction | a - b |
-| Multiplication | a * b |
-| Division |  a / b |
-| Modulo/Remainder | a % b |
-| Exponentiation | a ^ b |
-| Factorial | x ! |
-| Sine | sin( x ) |
-| Cosine | cos( x ) |
-| Tangent | tan( x ) |
-| Arc Tangent | arctan( x ) |
+| Operation |  Operator Syntax | Description |
+| :- | :- | :- |
+| Addition |  a + b | |
+| Subtraction | a - b | |
+| Multiplication | a * b | |
+| Division |  a / b | |
+| Modulo/Remainder | a % b | |
+| Exponentiation | a ^ b | |
+| Ceil | ceil( x.y ) |  Rounds a number up to the next largest integer. |
+| Floor | floor( x.y ) | Rounds a number down to the next smallest integer. |
+| Exponential function | exp( x ) | Returns e raised to the specified power. |
+| Maximum | max( x, y, z ) | |
+| Minimum | max( x, y, z ) | |
+| Absolute | abs( - x ) | Absolute value of a number |
+| Log10 | log( x ) | |
+| Log base e | ln( x ) | |
+| Square root | sqrt( x ) | |
+| Power | pow( x, y ) | Calculate a number raised to the power of some other number. |
+| Factorial | x ! | |
+| Sign | sign( - x ) | A number that indicates the sign of value:<br />- `-1` if number is less than zero.<br />- `0` if number is zero.<br />- `1` if number is greater than zero. |
+| Round | round( x.abcd ) | | Example: `round(8.7867)`
+| Random | rand() | |
+| Pi | +pi | Returns the number of Pi. |
+| Sine | sin( x ) | |
+| Cosine | cos( x ) | |
+| Tangent | tan( x ) | |
+| Arc Sine | arcsin( x ) | |
+| Arc Cosine | arccos( x ) | |
+| Arc Tangent | arctan( x ) | |
+| Hyperbolic Sine | sinh( x ) | |
+| Hyperbolic Cosine | cosh( x ) | |
+| Hyperbolic Tangent | tanh( x ) | |
+| Hyperbolic Arc Sine | arsinh( x ) | |
+| Hyperbolic Arc Cosine | arcosh( x ) | |
+| Hyperbolic Arc Tangent | artanh( x ) | |
```

Comment:
`artanh( x )` isn't woking at the moment. See https://github.com/microsoft/PowerToys/issues/17323 for more details.

Reply:
Show I add a hint in the third column?

Reply:
> Show I add a hint in the third column?

I think the people who need this knows how to user it.

Reply:
I think not. That's part of the mathematical understanding of what the function represents. It'd be like adding a note to square root saying it doesn't accept negative numbers because the square root of negative numbers doesn't exist in real numbers.